hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlTable.java

```diff
@@ -116,6 +116,11 @@ protected void createTable(Session session, TableDefine tableDefine) {
         }
     }
 
+    protected String engine(Session session) {
```

Comment:
also update signature of [PostgresqlTable.engine()](https://github.com/hugegraph/hugegraph/blob/master/hugegraph-postgresql/src/main/java/com/baidu/hugegraph/backend/store/postgresql/PostgresqlTable.java#L49)